### ISSUE TITLE
🐜 [Bug][1.6.1]Sometimes the boat  from Menethil harbor to Auberdine would stop sailing beforing the map transfer to Kalimdor

### ISSUE THREAD

**Issue by den13501** [vmangos/core#1254]
### Expected behavior

boad should be sailing between  Auberdine and Menethil harbor without problem
### Steps to reproduce

1. type command `.tele menethil` and go to the right side deck.
2. taking the boat that from Auberdine habor to Menethil harbor.
3. watch the behavior of the boad, sometimes it stop sailing  beforing the map transfer to Kalimdor.
4. Video
[![recording](https://external-preview.redd.it/EEboeiECTaPoPYTPJaoLtupCkiFwUaStgg2o0KBxDfg.jpg?auto=webp&s=82dce1446688dcd97c28a1889b457f98186aef04)](https://streamable.com/regkdh)
snapshot1 - the boad stop in dalaran
![image](https://user-images.githubusercontent.com/58270741/123876718-65a6eb00-d96e-11eb-99ac-906a2409af18.png)
snapshot2 - the boad stop in dalaran and the texture become broken 
![image](https://user-images.githubusercontent.com/58270741/123876770-7bb4ab80-d96e-11eb-9b64-6977304f5f00.png)

### Version & Environment

Client Version: 1.6.1.4544
CLIENT_BUILD_1_6_1
Server Patch: 1.2

Commit Hash:  

OS Client: Windows 10
OS Server: Windows server 2008 r2

### Crashlog

- None

**Comment by ratkosrb** [comment]
If it still happens on the latest version then I don't know how to fix it.

**Comment by den13501** [comment]
> If it still happens on the latest version then I don't know how to fix it.

Yes, it still happens on latest version, an I found sometime the boat will  become normal after you just move around on the boat

**Comment by ratkosrb** [comment]
Revert to revision before elevator transports were implemented or use another emulator.

```
git reset --hard 2368e0470ce2b22d1c527172ab48c7cf4660c81d
```

**Comment by GrenderG** [comment]
Maybe it's only happening in 1.6.1? No player reported issues on Kalidar since the latest fixes.

**Comment by den13501** [comment]
Yes , I just try reproduce same steps on server 1.12(patch 1.2), 1.9(patch 1.2) ,1.7(patch 1.2 ), everything is fine..
So, if it is unavailable to fix for this 1.6, I will close this issue . I think I will make a teleporter to be a temporary solution ^_^a

**Comment by den13501** [comment]
add more discover, sometimes the boat will let me wait for a long time, you can see the boat in a far way, but they suddently dispear

**Comment by ghost** [comment]
Issue still happening
